### ISSUE TITLE
Escape prevent construction does not working with empty string

### ISSUE THREAD

**Issue by a-x-** [bem/bem-xjst#321]
```js
BEMHTML.apply({html: ''})
// --> "[object Object]"
```

**Comment by Yeti-or** [comment]
```
var bemxjst = require('bem-xjst');
// "version": "7.0.2"
var bemhtml = bemxjst.bemhtml;
var templates = bemhtml.compile();
console.log(templates.apply({ html: '' }));
// [object Object]
```

**Comment by miripiruni** [comment]
Fixed in v7.0.3 and ported to v6.5.5.
